sci-physics/hepmc/hepmc-3.3.0-r1.ebuild

```diff
@@ -0,0 +1,72 @@
+# Copyright 2022-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..12} )
+inherit fortran-2 cmake python-single-r1
+
+MYP=HepMC3-${PV}
+
+DESCRIPTION="Event Record for Monte Carlo Generators"
+HOMEPAGE="https://hepmc.web.cern.ch/hepmc/"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://gitlab.cern.ch/hepmc/HepMC3"
+else
+	SRC_URI="https://hepmc.web.cern.ch/hepmc/releases/${MYP}.tar.gz"
+	S="${WORKDIR}/${MYP}"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-3+"
+SLOT="3"
+IUSE="doc test examples python root static-libs"
+RESTRICT="!test? ( test )"
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+# Automagic comrpession dependency will hopefully be removed in upstream soon.
+# https://gitlab.cern.ch/hepmc/HepMC3/-/issues/99
+# For now we install all of them...
+RDEPEND="
+	sys-libs/zlib
+	app-arch/xz-utils
+	app-arch/bzip2
+	app-arch/zstd
+	python? (
+		${PYTHON_DEPS}
+		$(python_gen_cond_dep '
+			dev-python/numpy[${PYTHON_USEDEP}]
+		')
+	)
+"
+DEPEND="${RDEPEND}"
+BDEPEND="
+	root? ( sci-physics/root:= )
+	doc? (
+		app-text/doxygen[dot]
+		dev-texlive/texlive-latex
+		dev-texlive/texlive-latexextra
+		dev-texlive/texlive-latexrecommended
+	)
+"
+
+src_configure() {
+	local mycmakeargs=(
+		-DHEPMC3_PYTHON_VERSIONS="${EPYTHON/python/}"
+		-DHEPMC3_ENABLE_ROOTIO=$(usex root ON OFF)
+		-DHEPMC3_ENABLE_PYTHON=$(usex python ON OFF)
+		-DHEPMC3_ENABLE_TEST=$(usex test ON OFF)
+		-DHEPMC3_BUILD_DOCS=$(usex doc ON OFF)
+		-DHEPMC3_BUILD_EXAMPLES=$(usex examples ON OFF)
+		-DHEPMC3_BUILD_STATIC_LIBS=$(usex static-libs ON OFF)
```

Comment:
Are the static libs useful for anything? Usually we disable them unconditionally.

Reply:
I'd leave the option in. In simulation, static libraries are often used to get better [performance](https://cds.cern.ch/record/2905024/files/ATL-SOFT-SLIDE-2024-257.pdf#page=5&view=FitH), it's a valid use case. Cheers,